crates/rolldown_binding/src/utils/normalize_binding_options.rs

```diff
@@ -83,7 +83,6 @@ pub fn normalize_binding_options(
   >,
   #[cfg(not(target_family = "wasm"))] worker_manager: Option<WorkerManager>,
 ) -> napi::Result<NormalizeBindingOptionsReturn> {
-  debug_assert!(PathBuf::from(&input_options.cwd) != PathBuf::from("/"), "{input_options:#?}");
   let cwd = PathBuf::from(input_options.cwd);
```

Comment:
The "rollup@function@relative-outside-external: correctly resolves relative external imports from outside directories" test is allow  `cwd: /`